Makefile

```diff
@@ -158,11 +159,15 @@ smokecheck: clean develop
 	./scripts/avocado run passtest.py
 
 check: clean develop check_cyclical modules_boundaries
-	selftests/checkall
+	AVOCADO_CHECK_LEVEL=0 selftests/checkall
```

Comment:
Would you have anything against keeping `make check` as default, without setting the `AVOCADO_CHECK_LEVEL`? The reason is I'd prefer keep using `make check` while setting `AVOCADO_CHECK_LEVEL=99` in `.bashrc` and this would override it.

If you dislike it, how about having `make check` which runs default set, `make check-short` which forces to 0 and the rest as defined here?

Alternatively I'd simply skip the `check-*` targets and keep only `make check` and leave it on users to define the env variable.